examples/templates/docker/main.tf

```
@@ -8,17 +8,36 @@ terraform {
       source  = "kreuzwerker/docker"
       version = "~> 3.0.1"
     }
+    external = {
+      source  = "hashicorp/external"
+      version = "~> 2.3.1"
+    }
   }
 }
 
 locals {
-  username = data.coder_workspace.me.owner
+  username        = data.coder_workspace.me.owner
+  host_os         = data.coder_provisioner.me.os
+  unix_command    = <<EOF
+  docker context inspect $(docker context show) | grep Host | awk '{print $2}' | tr -d '"' | tr -d ',' | tr -d '\n' | jq -R | jq '{DOCKER_HOST: .}'
```

I'm not sure about making our default template dependent on `jq`, seems like a hindrance to get started. 🤔

If we end up using jq anyway, might I suggest a simplification?

```suggestion
  docker context inspect $(docker context show) | jq '{DOCKER_HOST: .[].Endpoints.docker.Host}'
```

Here's an almost "zero dependency" command:

```shell
docker context inspect $(docker context show) | sed -ne 's/.*"Host":\(.*\),$/{"DOCKER_HOST":\1}/p'
```